test/runtests.jl

```diff
@@ -10,19 +10,19 @@ solvers = [
            ODE.ode2_midpoint,
            ODE.ode2_heun,
            ODE.ode4,
-           ODE.ode4ms,
-           ODE.ode5ms,
+           # Ode.ode4ms,
```

Comment:
Great, I'm at the JuliaCon as well, how about we meet in the lobby after this lecture?